Source/Core/Core/Movie.h

```diff
@@ -52,6 +52,8 @@ struct ControllerState
 static_assert(sizeof(ControllerState) == 8, "ControllerState should be 8 bytes");
 #pragma pack(pop)
 
+// When making changes to the DTM format, keep in mind that there are programs other
+// than Dolphin that parse DTM files. The format is expected to be relatively stable.
```

Comment:
Just to be explicit, please mention something about order.

Reply:
I think that would be unnecessary clutter. The ordering of data is a part of any binary header format.